src/personalData/bin/www

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env node
```

Comment:
@cindyli  FYI, I remembered the express server creation tool that made this script: it's the "[Express application generator](https://expressjs.com/en/starter/generator.html)".  It not only generated the `www` script, but a complete directory structure. That structure is shown near the bottom of the page at that link.  Also, as that page states, (paraphrase) there is no requirement to keep it this.  I tweaked things a little as the development progressed.

I don't know if that helps other than to provide some "documentation" (I use the term lightly) on why certain files exist and why the structure is what it is.  I hope it's useful.